### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- Travis CI -> GitHub Actions
- upgraded composer pacxkages
- polished docs

https://github.com/contributte/contributte/issues/47

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,17 +17,18 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": ">7.2",
```
Comment (f3l1x) [inline composer.json]: `>=`

Comment (petrparolek) [reply composer.json]: thanks

**Comment by f3l1x** [comment]
@petrparolek Update composer `>=7.2` and I will merge it.

**Comment by petrparolek** [comment]
thanks fixed

**Pull request merged**